seastar/src/sleep.rs

```diff
@@ -7,5 +7,38 @@ mod ffi {
 
         #[namespace = "seastar_ffi"]
         type VoidFuture = crate::cxx_async_futures::VoidFuture;
+
+        fn sleep_secs(duration: i32) -> VoidFuture;
+        fn sleep_millis(duration: i32) -> VoidFuture;
+        fn sleep_micros(duration: i32) -> VoidFuture;
+        fn sleep_nanos(duration: i64) -> VoidFuture;
     }
 }
+
+/// Returns a future which completes after a specified time in seconds has elapsed.
+///
+/// Uses steady clock defined in Seastar.
+pub async fn sleep_secs(duration: i32) {
+    ffi::sleep_secs(duration).await.unwrap();
+}
+
+/// Returns a future which completes after a specified time in milliseconds has elapsed.
+///
+/// Uses steady clock defined in Seastar.
+pub async fn sleep_millis(duration: i32) {
+    ffi::sleep_millis(duration).await.unwrap();
+}
+
+/// Returns a future which completes after a specified time in microseconds has elapsed.
+///
+/// Uses steady clock defined in Seastar.
+pub async fn sleep_micros(duration: i32) {
+    ffi::sleep_micros(duration).await.unwrap();
+}
+
+/// Returns a future which completes after a specified time in nanoseconds has elapsed.
+///
+/// Uses steady clock defined in Seastar.
+pub async fn sleep_nanos(duration: i64) {
+    ffi::sleep_nanos(duration).await.unwrap();
+}
```

Comment:
Please prefix those with `steady_` - just like you did with `lowres_` functions, so that it is clear which timer those functions are using.

Reply:
Done.